mmdeploy/codebase/base/task.py

```diff
@@ -176,10 +176,10 @@ def _merge_cfg(cfg):
                     ' field of config. Please set '\
                     '`visualization=dict(type="VisualizationHook")`'
 
-                cfg.default_hooks.visualization.enable = True
+                cfg.default_hooks.visualization.draw = True
                 cfg.default_hooks.visualization.show = show
                 cfg.default_hooks.visualization.wait_time = wait_time
-                cfg.default_hooks.visualization.out_dir = show_dir
+                cfg.default_hooks.visualization.test_out_dir = show_dir
```

Comment:
is it really necessary to change here?

Reply:
Yes. If not I can not pass mmyolo model visualization

Reply:
then  mmyolo should change to align with most codebases @hhaAndroid